stable/fluent-bit/templates/config.yaml

```diff
@@ -14,7 +14,6 @@ data:
         Flush        {{ .Values.service.flush }}
         Daemon       Off
         Log_Level    {{ .Values.service.logLevel }}
-        Parsers_File parsers.conf
```

Comment:
What is the ramification of this being removed?

Reply:
Please check the PR #15775 for details on that.